.gitignore

```diff
@@ -1,4 +1,9 @@
 ERROR_LOG
 DEBUG_LOG
 imagecache/
-images/
\ No newline at end of file
+images/
+
+# Prevent local collection and configuration from being uploaded to git
```

Comment:
Could we pull these out into a separate commit?

Reply:
Of course I can do it. :-)